### PR TITLE
:FC-476: :speech_balloon: Update permission table Faas

### PR DESCRIPTION
**Pull request by agalagade**

### CODE REVIEW

File: pages/Documents/DeveloperTools/LivePersonFunctions/getting-started.md [hunk 1]
```diff
@@ -141,6 +145,20 @@ There are 3 user permissions pertaining to Functions:
     <td>✅</td>
     <td>✅</td>
   </tr>
+  <tr>
+    <td>logs</td>
+    <td>write</td>
+    <td>✅</td>
+    <td>-</td>
+    <td>-</td>
+  </tr>
```
Comment (LivePersonFaas) [inline pages/Documents/DeveloperTools/LivePersonFunctions/getting-started.md:154]: log read is missing

**Review by LivePersonFaas — CHANGES_REQUESTED** [review]

File: css/getting-started.css [hunk 2]
```diff
@@ -254,4 +254,26 @@
...
+    visibility: hidden;
+    width: 120px;
+    background-color: black;
+    color: #fff;
+    text-align: center;
+    padding: 5px 0;
+    border-radius: 6px;
+    position: absolute;
+    z-index: 1;
+    bottom: 100%;
```
[the remark this answers is not in the document]
Comment (agalagade) [reply css/getting-started.css]: removed

File: pages/Documents/DeveloperTools/LivePersonFunctions/getting-started.md (hunk 1 above)
Comment (agalagade) [reply pages/Documents/DeveloperTools/LivePersonFunctions/getting-started.md:157]: added

File: pages/Documents/DeveloperTools/LivePersonFunctions/getting-started.md [hunk 3]
```diff
@@ -101,7 +101,11 @@ There are 3 user permissions pertaining to Functions:
   </tr>
   <tr>
     <td>secret</td>
-    <td>read plain</td>
+    <td>
+    <div class="tooltips">read
+        <span class="tooltiptext">secret value is obfuscated</span>
+    </div>
+    </td>
```
[the remark this answers is not in the document]
Comment (agalagade) [reply pages/Documents/DeveloperTools/LivePersonFunctions/getting-started.md]: added

File: pages/Documents/DeveloperTools/LivePersonFunctions/getting-started.md [hunk 4]
```diff
@@ -101,7 +101,11 @@ There are 3 user permissions pertaining to Functions:
   </tr>
   <tr>
     <td>secret</td>
-    <td>read plain</td>
+    <td>
+    <div class="tooltips">read
+        <span class="tooltiptext">secret value is obfuscated</span>
```
[the remark this answers is not in the document]
Comment (agalagade) [reply pages/Documents/DeveloperTools/LivePersonFunctions/getting-started.md]: updated

**Comment by lmartliveperson** [comment]
Please verify post merge

**Pull request merged**